posthog/queries/trends/sql.py

```diff
@@ -167,15 +167,15 @@
     {aggregate_operation} as total, day_start, breakdown_value
 FROM (
     SELECT any(session_duration) as session_duration, day_start, breakdown_value FROM (
-        SELECT $session_id, session_duration, {interval_annotation}(timestamp, {start_of_week_fix} %(timezone)s) as day_start,
+        SELECT sessions.\"$session_id\", session_duration, {interval_annotation}(timestamp, {start_of_week_fix} %(timezone)s) as day_start,
```

Comment:
This looks solid - only q I have is should this be hardcoded to `sessions` or should you use `SessionQuery.SESSION_TABLE_ALIAS` which seems to be a constant with that value?

Reply:
good point

Reply:
tbh don't really like that sessions is the table that works here because sessions join is conditional. Still puzzled why "events" fails sometimes

Reply:
yeah, definitely should not be sessions here.

re: events, I _suspect_ it failed the first time because we were missing the double quotes? Some issue with `$` columns I recollect.

Reply:
I tried all variants of naming the column with events and it doesn't seem to work consistently for both cases